src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php

```diff
@@ -1025,6 +1027,45 @@ protected function createChildContext(array $parentContext, string $attribute, ?
 
         $this->assertFalse($normalizer->childContextCacheKey);
     }
+
+    /**
+     * @see https://github.com/symfony/symfony/issues/54378
```

Comment:
We usually do not add GitHub links in the code

Reply:
Thanks @OskarStark, removed.